src/server/src/rollbar/rollbar-exception.filter.ts

```diff
@@ -20,6 +26,18 @@ function isWhitelisted(exception: HttpException) {
   );
 }
 
+function parseIp(req: IGetUserAuthInfoRequest): string | undefined {
+  if (req.headers["x-forwarded-for"]) {
+    if (Array.isArray(req.headers["x-forwarded-for"])) {
+      return [...req.headers["x-forwarded-for"]].shift();
```

Comment:
Is this different from the following?
```suggestion
      return req.headers["x-forwarded-for"][0];
```

Reply:
There is no guarantee that the index has at least one element, probably safer to use shift

Reply:
Playing around in the Chrome JS console shows no difference in that case:
```
>> a = []
⬅ Array []

>>a.shift()
⬅ undefined
>>a[0]
⬅ undefined

​```